recipes/slow-buffer-to-buffer-alloc-unsafe-slow/README.md

```diff
@@ -0,0 +1,58 @@
+# SlowBuffer to Buffer.allocUnsafeSlow Codemod
+
+This codemod migrates deprecated `SlowBuffer` usage to `Buffer.allocUnsafeSlow()` to handle Node.js DEP0030.
+
+## What it does
+
+This codemod transforms:
+
+1. `SlowBuffer` constructor calls to `Buffer.allocUnsafeSlow()`
+2. Direct `SlowBuffer` calls to `Buffer.allocUnsafeSlow()`
+3. Import/require statements to use `Buffer` instead of `SlowBuffer`
```

Comment:
```suggestion
-`SlowBuffer` constructor calls to `Buffer.allocUnsafeSlow()`
- Direct `SlowBuffer` calls to `Buffer.allocUnsafeSlow()`
- Import/require statements to be synced with new function
```

it's different point there are any kind order. 

Therefore, it does not necessarily transform `Buffer` into `SlowBuffer`; it depends on the case.